### DISCUSSION TITLE
Integrate check_date and check_date:opening_hours tags for POI

### DISCUSSION THREAD

**Discussion by elliotsjoqvist** [osmandapp/OsmAnd#20727] [Ideas]
When looking at a POI in osmand there is no information about how old the data is. I think it's time for osmand to make use of the check_date tags. Especially for opening hours it should be written below something like "last updated: date" or "updated X weeks ago". 

It can even have color schemes like green is recently updated and red for a several months old on the check_date text.

1.6 million objects has a check_date tag
[https://taginfo.openstreetmap.org/keys/check_date#overview](url)

160 thousand for check_date:opening_hours
[https://taginfo.openstreetmap.org/keys/check_date%3Aopening_hours#overview](url)

**Comment by mariush444** [comment]
https://github.com/osmandapp/OsmAnd/issues/20083
